README.md

```diff
@@ -63,43 +61,65 @@ from `CAM/cime/scripts/`.
 
 You can then navigate to the case directory at `<path_to_testcase_directory>`.
 
-#### Building with _FTorch_
+### Build CAM with `FTorch`
 
-To couple to FTorch modify `Tools/Makefile` line 600 to set the environment variable
-FTORCH_LIB to the location of the FTorch library on your system.
+Before we can run `./case.build` we first need to make some manual changes to the `Makefile` located in
+`path_to_testcase_directory>/Tools/Makefile`. This will allow the CIME build system to locate `FTorch`.
 
-On Derecho this is `/path/to/ftorch/bin/ftorch_intel` as defined below in
-[_FTorch_ on Derecho](#ftorch-on-derecho).
+From the test case directory, modify `Tools/Makefile` line 602 to set the environment variable `FTORCH_LIB` to the location of
+the `FTorch` library on your system.
 
-#### Setting up case details
+If you followed the instructions in section [`FTorch` on Derecho](#ftorch-on-derecho) this will be `$HOME/FTorch/bin/ftorch_intel`.
 
-We can now run `./case.setup` from within the case directory.
-Once this has been done then edit the generated `user_nl_cam` in the case directory
-as required.
-Add the following lines:
+```make
+FTORCH_LIB := $HOME/FTorch/bin/ftorch_intel
+```
 
-#. `gw_convect_dp_ml='on'`\
-    This is the switch to use our new ML convective-gw scheme instead of the default.\
-    Other options are `'off'` (default - use original), `'bothoff'` (run both schemes
-    but use default for simulation), and `'bothon'` (run both but use ML for simulation).
-#. `gw_convect_dp_ml_net='<PATH/TO/MODEL.pt>'`\
-    The path to the your saved PyTorch model.
+### Setting up case details
 
-Also consider adding:
-```
-fincl<n> = 'MYVAR'
+We can now run `./case.setup` from within the case directory. Once this has been done then edit the generated `user_nl_cam` in
+the case directory as required. Add the following lines:
+
+```fortran
+gw_convect_dp_ml=.false.
+gw_convect_dp_ml_compare=.true.
+gw_convect_dp_ml_net_path='/path/to/neural/net'
+gw_convect_dp_ml_norms='/path/to/norms'
 ```
```

Comment:
This is an example configuration, but users can vary for their applications.
Adding the explanations below is really helpful, thanks @TomMelt but maybe shoudl add a note to say that this config is an example and can be tailored to your particular experiment as per the descriptions below?

Reply:
I have added some more wording [here](https://github.com/DataWaveProject/CAM/blob/f8c4ae9d6466d2631e998396ba86e6944f2838d7/README.md). lmk what you think

Reply:
Yeah, seems good.